### PR TITLE
ttt-utils:0.1.0

### PR DESCRIPTION
**Pull request by jomaway**
I am submitting
- [x] a new package

Description: Collection of tools which makes a teachers life easier, assignment creation, notebook fields, ...
Will be the core package for some templates i want to publish soon.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

**Review by reknih — COMMENTED** [review]
Just two comments on the README

File: packages/preview/ttt-utils/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+#import assignments: assignment, question, answer
+#import components: caro as grid_pattern
+
+// Add a question.
+
+#assignment[First assignment
+
+    #question[
+        #answer(field: grid_pattern(5))
```
[the remark this answers is not in the document]
Comment (jomaway) [reply packages/preview/ttt-utils/0.1.0/README.md]: ah true, forgot to update this.

File: packages/preview/ttt-utils/0.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+]
+
+```
+
+See the [docs](./docs/docs.typ) for all functions and features.
+
+
+## CHANGELOG 
+
+See [CHANGELOG.md](../CHANGELOG.md)
```
[the remark this answers is not in the document]
Comment (jomaway) [reply packages/preview/ttt-utils/0.1.0/README.md]: Yes, removed the links as those are not finished yet anyway. But will use absolute links next time

**Comment by jomaway** [comment]
@reknih should be resolved now.

**Comment by reknih** [comment]
Thank you!

**Pull request merged**